app/schemas.py

```diff
@@ -274,7 +274,7 @@ class Meta(BaseSchema.Meta):
     def validate_permissions(self, value):
         permissions = [v.permission for v in value]
         for p in permissions:
-            if p not in models.SERVICE_PERMISSION_TYPES:
+            if p not in models.SERVICE_PERMISSION_TYPES and p != "international_letters":
```

Comment:
Good catch on this permission! This is one that we shouldn't be using anywhere in the application, so could we check whether we can go the other direction here and remove it from the total set of permissions?

Reply:
Okay.  I have removed it.  The audit for this PR is failing right now because of the requests package, but there is a dependabot PR for that.

Reply:
Also, I don't know why -- when I push additional commits -- the review doesn't update.  Right now it is still just showing the original change, but I pushed a db migration etc.